src/components/external/DisputeManagement/components/DefendDisputeFlow/constants.ts

```diff
@@ -0,0 +1,13 @@
+const UPLOAD_DOCUMENT_MAX_SIZE = {
+    ['2MB']: 2097152,
+    ['10MB']: 10485760,
```

Comment:
Just curious as to the usage of computed prop syntax here, instead of:
```suggestion
    '2MB': 2097152,
    '10MB': 10485760,
```